Changelog.md

```diff
@@ -1,5 +1,47 @@
 # Changelog
 
+### 1.1.2
+
+**Image:** 
+- k8s.gcr.io/ingress-nginx/controller:v1.1.1@sha256:28b11ce69e57843de44e3db6413e98d09de0f6688e33d4bd384002a44f78405c
```

Comment:
the tag version says 1.1.1

Reply:
oh shoot. fixing it now

Reply:
@strongjz  Sorry for confusing changes to PRs. Please check again. Pushed a new commit